lib/services/pool/lib/util.ts

```diff
@@ -570,3 +573,77 @@ export function poolGetNestedTokenEstimateForPoolTokenAmounts({
 export function tokenAmountsAllZero(tokenAmounts: TokenAmountHumanReadable[]) {
     return tokenAmounts.filter((amount) => parseFloat(amount.amount) === 0).length === tokenAmounts.length;
 }
+
```

Comment:
Any reason for these functions to not live in `PoolProportionalInvestService`?

Reply:
No reason anymore. I have moved them to the service.